.npmignore

```diff
@@ -7,12 +6,16 @@ src
 tasks
 test
 tmp
+.babelrc
 .editorconfig
-.gitignore
 .eslintrc
-.jscsrc
+.gitignore
 .travis.yml
 bower.json
 CONTRIBUTING.md
 gulpfile.babel.js
+ISSUE_TEMPLATE.md
+marionette-logo.png
+PULL_REQUEST_TEMPLATE.md
+trigger-deploy-mn-com.js
 upgradeGuide.md
```

Comment:
I found this in the  [NPM documentation](https://docs.npmjs.com/misc/developers).
![image](https://cloud.githubusercontent.com/assets/8260678/18476305/7f3793da-7997-11e6-8477-0b2163ccb9ba.png)